app/models/miq_database.rb

```diff
@@ -32,11 +32,22 @@ def self.registration_default_values
   end
 
   def self.registration_default_value_for_update_repo_name
-    "cf-me-5.6-for-rhel-7-rpms rhel-server-rhscl-7-rpms"
+    Vmdb::Settings.template_settings.product.update_repo_names.to_a.join(" ")
   end
 
   def update_repo_names
-    update_repo_name.split
+    Settings.product.update_repo_names.to_a
+  end
+
+  def update_repo_name
+    update_repo_names.join(" ")
+  end
+
+  def update_repo_name=(repos)
+    return unless repos
+    hash = {:product => {:update_repo_names => repos.split}}
+    Vmdb::Settings.save!(MiqRegion.my_region, hash)
+    Settings.reload!
```

Comment:
IF the save! is not doing this we need an alternate method that propogates the changes to other servers.